_episodes/02-practice-learning.md

```diff
@@ -49,22 +49,22 @@ who applied the [Dreyfus model of skill acquisition][wikipedia-dreyfus-skill]
 in her studies of
 [how nurses progress from novice to expert](http://journals.sagepub.com/doi/10.1177/0270467604265061)
 ([see also books by Benner](https://www.worldcat.org/search?q=au%3ABenner%2C+Patricia+E.&qt=hot_author)). This work indicates that
-through practice and formal instruction, learners acquire skills and advance through distinct stages. In simplified form,
-the three stages of this model are:
+through practice and formal instruction, learners acquire skills and advance through distinct stages. In simplified form, three stages of this model are:
 
 ![Three people, labeled from left to right as "Novice", "Competent Practitioner", and "Expert". Underneath the people,
 an arrow labelled "Experience level" points from left to right. Underneath the figure labelled "Novice" a quote says "I'm not sure what questions to ask." The Competent Practitioner says "I'm pretty confident, but I still look stuff up a lot!" The Expert says "I've been doing this on a daily basis for years!"](../fig/skill-level.svg)
 
 *   *Novice*: someone who does not know what they do not know, i.e.,
     they do not yet know what the key ideas in the domain are or how they relate.
-    One sign that someone is a novice is that their questions "are not even wrong".
+    Novices may have difficulty formulating questions, or may ask questions that seem irrelevant or off-topic
+    as they reach for anything they know to connect.
```

Comment:
```suggestion
    Novices may have difficulty formulating questions, or may ask questions that seem irrelevant or off-topic
    as they rely on prior knowledge, without knowing what is/not related yet.
```

"reach for anything they know how to connect was vague, but i think this captures?